pages/contribute.vue

```diff
@@ -1,60 +1,87 @@
 <template>
   <div class="page">
-    <HeroBanner color="blue"><h1><span>How You Can Help</span></h1></HeroBanner>
+    <HeroBanner color="blue"
+      ><h1><span>How You Can Help</span></h1></HeroBanner
+    >
     <h3 class="intro">Contributor Guidelines</h3>
-    <div>
-      <h2>Thank you for your interest in contributing to LeaderboardsGG. We're glad you found your way here, and we're always in need of new contributors.</h2>
+    <div class="container">
+      <h2>
+        Thank you for your interest in contributing to LeaderboardsGG. We're
+        glad you found your way here, and we're always in need of new
+        contributors.
+      </h2>
 
       <h2>General Information</h2>
-      <p>We currently have three repositories set up. Below are the links to each one along with some basic information regarding their respective technology stacks. Make sure to read the contributing guidelines for the repository you wish to contribute to. This will ensure a smooth experience for all involved.</p>
+      <p>
+        We currently have three repositories set up. Below are the links to each
+        one along with some basic information regarding their respective
+        technology stacks. Make sure to read the contributing guidelines for the
+        repository you wish to contribute to. This will ensure a smooth
+        experience for all involved.
+      </p>
 
       <h2>Our Repositories</h2>
 
       <h3>Front End</h3>
       <ul>
+        <li>Technologies: Vue.js, Nuxt.js, Jest, TailwindCSS</li>
         <li>
-          Technologies: Vue.js, Nuxt.js, Jest, TailwindCSS
+          <a href="https://github.com/leaderboardsgg/leaderboard-site"
+            >Github Repo</a
+          >
         </li>
         <li>
-          <a href="https://github.com/leaderboardsgg/leaderboard-site">Github Repo</a>
+          <a
+            href="https://github.com/leaderboardsgg/leaderboard-site/blob/main/CONTRIBUTING.md"
+            >Contributing Guidelines</a
+          >
         </li>
         <li>
-          <a href="https://github.com/leaderboardsgg/leaderboard-site/blob/main/CONTRIBUTING.md">Contributing Guidelines</a>
-        </li>
-        <li>
-          <a href="https://github.com/leaderboardsgg/leaderboard-site/issues">Issues</a>
+          <a href="https://github.com/leaderboardsgg/leaderboard-site/issues"
+            >Issues</a
+          >
         </li>
       </ul>
 
       <h3>Back End</h3>
       <ul>
+        <li>Technologies: C#, ASP.NET Core, .NET 6.0, PostgreSQL</li>
```

Comment:
I thought we were on .NET 7.0? That's at least what it says in the [backend readme](https://github.com/leaderboardsgg/leaderboard-backend#visual-studio-codeother-editors)

Reply:
Updated